go.mod

```diff
@@ -1,52 +1,53 @@
 module github.com/aws/aws-network-policy-agent
 
-go 1.21.11
+go 1.22.4
 
 require (
 	github.com/aws/amazon-vpc-cni-k8s v1.18.1
 	github.com/aws/aws-ebpf-sdk-go v1.0.9
 	github.com/aws/aws-sdk-go v1.50.30
-	github.com/go-logr/logr v1.4.1
+	github.com/emilyhuaa/policyLogsEnhancement v1.0.2
```

Comment:
I'm a AWS CNI policy agent user and I was giving a look to this PR. I think adding this kind of dependency is a bad idea.

Reply:
@pviossat, thanks for taking a look. It's a PoC project we are experimenting on, ideally the dep should be hosted under some official org, and we need to figure it out (that's why we branch out this PR). But at this point we are testing on the functionality to improve the observability of NPA event logs.